src/linagora.esn.james/app/quota/user/james-quota-user.controller.spec.js

```diff
@@ -14,12 +14,6 @@ describe('The jamesQuotaUserController', function() {
 
   beforeEach(function() {
     angular.mock.module('linagora.esn.james');
-    angular.mock.module('esn.configuration', function($provide) {
```

Comment:
why we remove this ?

Reply:
Useless, we don't need to register the conf module here